.prettierrc

```diff
@@ -0,0 +1,8 @@
+{
+	"useTabs": true,
```

Comment:
@MateusKawazoe-ViaConsulting  Areyou agree with this formating ?

Reply:
Sure